lib/api_calls/organization.js

```diff
@@ -51,7 +51,7 @@ module.exports = stampit().
     removeMemberFromOrganization: function(params) {
```

Comment:
I wasn't aware until now that we also modified the function to remove a member from an organization. Should we document that in the mock-api repo?